crt_portal/cts_forms/templates/forms/complaint_view/show.html

```diff
@@ -0,0 +1,84 @@
+{% extends "forms/base.html" %}
+{% load static %}
+
+{% block page_header %}
+<header class="page-header-background">
+  <div class="grid-container">
+    <div class="grid-row flex-align-center">
+      <div class="grid-col-fill">
+        <h1>CRT Public Complaint Intake</h1>
+      </div>
+      <div class="grid-col-auto">
+        <a class="usa-button" href="/accounts/logout">Log out</a>
+      </div>
+    </div>
+  </div>
+</header>
+{% endblock %}
+
+{% block main_class %} class="page-header-background"{% endblock %}
+
+{% block content %}
+  <div class="grid-container">
+    <div class="complaint-header bg-primary text-white">
+      <div class="grid-container">
+      <div class="grid-row grid-gap">
+        <div class="tablet:grid-col-8">
+          <h2 class="text-uppercase">
+            Complaint <span class="text-gold">{{ data.id }}</span>
+          </h2>
+          <p>
+            <span class="text-gold text-uppercase">Received:  </span>
+            <span class="text-white">{{ data.create_date|date:'g:s a F j, Y' }}</span>
+          </p>
+        </div>
+        <div class="tablet:grid-col-3 tablet:grid-offset-1 text-right">
+          <a class="usa-button usa-button--light close-record" href="/form/view">
```

Comment:
Not sure if we want to make this part of this PR but I wanted to note that when we add sort and filter options, we will want to pass that to this template probably something like adding next={{ page_format.page_args }}&page={{ page_format.page }} to the url and then adding that to the close record link.

Reply:
Are both of these values present in the codebase now? Maybe we can go over the sorting changes that got merged yesterday? I'd love to get it into this PR and some assistance would be great 😄

Reply:
Happy to pitch in! would you like me to commit to this PR or make a PR targeting your PR?

Reply:
If it's not too much trouble, a commit here is great!

Reply:
added!